agent-core/src/main/java/com/newrelic/agent/android/analytics/EventManagerImpl.java

```diff
@@ -23,7 +23,7 @@
 public class EventManagerImpl implements EventManager, EventListener {
     private static final AgentLog log = AgentLogManager.getAgentLog();
     public static final int DEFAULT_MAX_EVENT_BUFFER_TIME = 600;    // 600 seconds (10 minutes)
-    public static final int DEFAULT_MAX_EVENT_BUFFER_SIZE = 4000;   // Beta testing value, to be revised with empirical data
+    public static final int DEFAULT_MAX_EVENT_BUFFER_SIZE = 1000;   // 1000 as the default
```

Comment:
Double check with Github Enterprise what the original comment is.

Reply:
This is the original comment --> // Beta testing value, to be revised with empirical data